src/Core/Helpers/AbstractConfigHelper.php

```diff
@@ -33,16 +33,22 @@ abstract class AbstractConfigHelper {
   const INVENTORY_ITEMS_PER_PAGE = 500;
 
   /**
+   * Retrieve the configuration client id for the global client
+   *
    * @return mixed
    */
   abstract public function getClientId();
 
   /**
+   * Retrieve the configuration client secret for the global client
```

Comment:
configured client secret for connections to Wayfair's secure APIs.
there is (unfortunately) more than one client using it.